usr/src/uts/common/vm/vm_page.c

```diff
@@ -5685,7 +5685,8 @@ page_migrate(
 	}
 }
 
-#define	MAX_CNT	60	/* max num of iterations */
+static uint_t reclaim_maxcnt = 60; /* max total iterations */
```

Comment:
Was just trying to understand the intent. If they are meant to be tunables you'll want to remove the static keyword. Though I haven't spent much time with the broader change yet.